parsl/tests/integration/test_stress/test_python_simple.py

```diff
@@ -1,16 +1,14 @@
-"""Testing bash apps
-"""
 import parsl
-from parsl import App, DataFlowKernel, ThreadPoolExecutor
+from parsl import python_app
+from parsl.configs.htex_local import config
+# from parsl.configs.local_threads import config
 
 import time
 import argparse
 
-workers = ThreadPoolExecutor(max_workers=100)
-dfk = DataFlowKernel(executors=[workers])
+parsl.load(config)
```

Comment:
In some cases, test modules get imported in workers (I think something to do with pytest adding on annotations, so probably not a problem with this test). In those cases, parsl.load() needs to happen in the protected `__main__` block, rather than everywhere that this file is imported, and it's probably stylistically better to try to stick to that for all tests. See c8c451b45c1b9e9163a2bf2a94ede74d9621fa91